doc/static/css/nrf.css

```diff
@@ -1,43 +1,6 @@
-/* background color top and bottom left */
-
-.wy-side-nav-search {
- background-color: #00a9ce;
-}
-
-.rst-versions .rst-current-version {
- background-color: #00a9ce;
-}
-
-/* Zephyr background color for the doc switcher */
-.rst-versions {
- background: #a67dfb;
```

Comment:
That one is now missing, which causes the scroll bar to be displayed in black (should probably be added in the theme).

Reply:
I'll check theme, thanks

Reply:
fixed

Reply:
This still doesn't work - the scroll bar is black.
(At least in Firefox)

Reply:
> Not sure where this is going wrong, but the doc set switcher doesn't display right for Zephyr:
> http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-4160/zephyr/index.html
> ![image](https://user-images.githubusercontent.com/11227796/111313880-27a4e600-8661-11eb-9295-eba4da56c0fe.png)

This is what I see on Firefox (and Chrome):
![image](https://user-images.githubusercontent.com/25011557/112005135-c4b0c480-8b22-11eb-8f28-9b893ed06bd6.png)

Reply:
I asked around in the techpubs team, and everybody sees a black scroll bar in Firefox. Might be on Windows only (if you're using Linux?).
Testing with http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-4160/nrf/index.html

Reply:
Tested on Windows, fixed the problem (actually scrollbar is now hidden unless necessary).

Reply:
Nice! Thanks. :)